packages/android_intent/example/lib/main.dart

```diff
@@ -39,14 +39,73 @@ class MyHomePage extends StatelessWidget {
     intent.launch();
   }
 
+  void _openGoogleMapsStreetView() {
+    final AndroidIntent intent = new AndroidIntent(
+        action: 'action_view',
+        data: Uri.encodeFull('google.streetview:cbll=46.414382,10.013988'),
+        package: 'com.google.android.apps.maps');
+    intent.launch();
+  }
+
+  void _displayMapInGoogleMaps({int zoomLevel: 12}) {
+    final AndroidIntent intent = new AndroidIntent(
+        action: 'action_view',
+        data: Uri.encodeFull('geo:37.7749,-122.4194?z=$zoomLevel'),
+        package: 'com.google.android.apps.maps');
+    intent.launch();
+  }
+
+  void _launchTurnByTurnNavigationInGoogleMaps() {
+    final AndroidIntent intent = new AndroidIntent(
+        action: 'action_view',
+        data: Uri.encodeFull(
+            'google.navigation:q=Taronga+Zoo,+Sydney+Australia&avoid=tf'),
+        package: 'com.google.android.apps.maps');
+    intent.launch();
+  }
+
+  void _openLinkInGoogleChrome() {
+    final AndroidIntent intent = new AndroidIntent(
+        action: 'action_view',
+        data: Uri.encodeFull('https://flutter.io'),
+        package: 'com.android.chrome');
+    intent.launch();
+  }
+
```

Comment:
Great examples, thanks!

Reply:
I've restructured the example a bit to have the explict intent test buttons grouped together in a separate view. It'll be easier for future tests to be added to the main view, instead of filling it up with explicit intent test buttons.